packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -105,6 +112,12 @@ class Dismissible extends StatefulWidget {
   /// has also been specified.
   final Widget secondaryBackground;
 
+  /// Gives the app an opportunity to confirm or veto a pending dismissal.
+  ///
+  /// If the returned Future<bool> completes true, then this widget will be
+  /// dismissed, otherwise it will be moved back to its original location.
```

Comment:
Would it make sense to explicitly say that a vetoed dismiss will not invoke onResize / onDismissed?

Reply:
Yes, good point.